### PR TITLE
chore: improve template pr with suggestion title and add rule

### PR DESCRIPTION
**Pull request by Natalie9**
## Infos

[Task](https://juntossomosmais.monday.com/boards/4069611459/pulses/6867486134)

#### What is being delivered?

Improved pull request and added PR title validation rule 

#### What impacts?

Pull Requests 

#### Reversal plan

Describe which plan we should follow if this delivery has to be reversed.

#### Evidences

Media(images, gifs or videos) that shows the result of your work.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 ## Infos
 
+<!--
+
+### Pull Request Title Convention
+
```
Comment (MayaraRMA) [inline .github/pull_request_template.md:6]: pq tá comentado?

File: .github/workflows/pr-title-check.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Validate PR Title
+        run: |
+          if [[ ! "${{ github.event.pull_request.title }}" =~ ^(feat|fix|chore|major)\([a-zA-Z0-9_-]+\):\ .+$ ]]; then
+            echo "O título do Pull Request deve seguir a convenção: 'tipo(contexto): descrição'."
+            echo "Os tipos devem ser: feat, fix, chore, major."
+            echo "Exemplo de título correto: feat(component): create link component"
```
Comment (felipefialho) [inline .github/workflows/pr-title-check.yml:20]: (block) the open source projets is always in english

File: .github/workflows/pr-title-check.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+name: PR Title Check
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
```
Comment (felipefialho) [inline .github/workflows/pr-title-check.yml:5]: we can run it just with opened pr to avoid unnecessary triggers

**Review by felipefialho — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-title-check.yml (hunk 2 above)
Comment (Natalie9) [reply .github/workflows/pr-title-check.yml]: https://github.com/juntossomosmais/atomium/pull/526/commits/537e8a3595e437a67a3e1da0c409343e0712c143

File: .github/workflows/pr-title-check.yml (hunk 3 above)
Comment (Natalie9) [reply .github/workflows/pr-title-check.yml]: https://github.com/juntossomosmais/atomium/pull/526/commits/537e8a3595e437a67a3e1da0c409343e0712c143

File: .github/pull_request_template.md (hunk 1 above)
Comment (Natalie9) [reply .github/pull_request_template.md:6]: It is commented to appear as an alert in the template during editing but not to be displayed in the final PR.

File: .github/workflows/pr-title-check.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  pr-title:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Validate PR Title
+        run: |
+          if [[ ! "${{ github.event.pull_request.title }}" =~ ^(feat|fix|chore|major)\([a-zA-Z0-9_-]+\):\ .+$ ]]; then
```
Comment (felipefialho) [inline .github/workflows/pr-title-check.yml:17]: We also need to add another convention to the title

```suggestion
          if [[ ! "${{ github.event.pull_request.title }}" =~ ^(feat|fix|chore|major|docs|ci)\([a-zA-Z0-9_-]+\):\ .+$ ]]; then
```

Comment (Natalie9) [reply .github/workflows/pr-title-check.yml]: https://github.com/juntossomosmais/atomium/pull/526/commits/d02ccf62710557500f7bc0b65ea588605b7f7ce7

**Review by felipefialho — APPROVED** [review]
🚀

**Pull request merged**